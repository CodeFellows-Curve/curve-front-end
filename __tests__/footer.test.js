import React from "react"
import { Provider } from 'react-redux'
import renderer from "react-test-renderer"
import Footer from "../src/components/layout/footer.js"
import configureStore from 'redux-mock-store'
import { mount, shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

const initialState = {}; 

const mockStore = configureStore();
let wrapper;
let store;

configure({ adapter: new Adapter() });

describe("Footer", () => {
    beforeEach(() => {
        //creates the store with any initial state or middleware needed  
        store = mockStore(initialState)
        wrapper = mount(<Provider store={store}><Footer /></Provider>)
    })

  it("Footer renders correctly", () => {
    const tree = renderer
      .create(<Footer store={store}/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})