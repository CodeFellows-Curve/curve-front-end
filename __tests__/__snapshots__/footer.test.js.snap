// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer renders correctly 1`] = `
<div
  className="sc-bdVaJa iCkKpr"
>
  <div>
    Â© 
    2019
     
    <span>
      Curve
    </span>
  </div>
</div>
`;
